### PR TITLE
Prepare releases

### PR DESCRIPTION
**Pull request by dvdplm**
kvdb: 0.2.0
kvdb-memorydb: 0.2.0
kvdb-rocksdb: 0.3.0
kvdb-web: 0.2.0
parity-util-mem: 0.3.0

### CODE REVIEW

**Comment by dvdplm** [comment]
> great, can we include `fixed-hash` and `primitive-types` too? (https://github.com/paritytech/parity-common/pull/268/files)

Ofc, can you suggest something for the CHANGELOG?

File: fixed-hash/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.5.2] - 2019-12-19
+### Fixed
+- re-export `alloc` for both std and no-std to fix compilation (See [PR #268](https://github.com/paritytech/parity-common/pull/268))
```
Comment (dvdplm) [inline fixed-hash/CHANGELOG.md]: @niklasad1 works for you?

File: fixed-hash/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fixed-hash"
-version = "0.5.1"
+version = "0.5.2"
```
Comment (dvdplm) [inline fixed-hash/Cargo.toml]: I don't think this is a breaking change, is it @ordian?

File: fixed-hash/CHANGELOG.md (hunk 1 above)
Comment (niklasad1) [reply fixed-hash/CHANGELOG.md]: :+1:

**Pull request merged**